styie.css

```diff
@@ -0,0 +1,43 @@
+h1{display: inline;
+padding-left: 35px;
+padding-right: 75px;
+}
+.topnav {
+    width: 100%;
+ background-color: antiquewhite;
+ border-style: none;
+ border-width: 2px;
+ float: left;
+ position: relative;
+ }
+ a:link, a:visited{
+     width: 100%;
+     background-color: antiquewhite;
+     padding: 5px 20px;
+     text-align: center;
+     text-decoration: none;
+     display: inline;
+     margin-left: 10px;
+ }
+ a:hover, a:active{
+     background-color: tomato;}
+.home {
+    height: 500px;
+    width: 100%;
+    background-image: url("food-processing.jpg");
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    margin :auto;
+}
+div.content {
+ height: 200px;
+ width: 400px;
+ padding-left: 50px;
+ padding-top: 50px;
+ background-color: white; 
+ opacity:0.8;
+ vertical-align: middle; 
+ position: absolute;
```

Comment:
it wont work as the parent position is static, so either remove it or need to make parent as relative or absolute positioned